packages/scss/src/components/_sample/component.scss

```
@@ -2,8 +2,9 @@
 // It does not contain its selector, which is present in the index.
 // Any sub-components are placed in the `@at-root`.
 // The `@at-root` eliminates this selector by default, and its parameter can be used to restore it.
+@use '@lucca-front/scss/src/commons/utils/namespace';
 
-@mixin component($atRoot: 'without: rule') {
+@mixin component($atRoot: namespace.$defaultAtRoot) {
```

```suggestion
@mixin component($atRoot: config.$atRoot) {
```

I'd like to see something like that here.

---

`$defaultAtRoot` is not configurable. Should it be moved in `config` anyway ?

---

It's not configurable, but it can be modified, so I found default in the name confusing on first reading.
`$globalAtRoot` perhaps?

In config, you have everything you need to configure LF, and if this configuration is modifiable by the BUs, the addition of `!default` at the end of the declarations.

So, to sum up, I'd say :
 - it goes in the config file
 - it doesn't take `!default`